tutorjupyterlite/patches/k8s-deployments

```diff
@@ -23,6 +23,9 @@ spec:
           volumeMounts:
             - name: fss-pvc-content
               mountPath: /jupyterlite/{{ JUPYTERLITE_CONTENT_DIR }}
+          env:
```

Comment:
Remove this now, as you already updated the Dockerfile.

Reply:
As discussed in meeting, we had to modify settings in both Dockerfile and K8s-deployments.
@wahajArbisoft @ziafazal